src/fragments/lib-v1/analytics/ios/record.mdx

```diff
@@ -42,6 +42,47 @@ Events have default configuration to flush out to the network every 60 seconds.
 
 > **Note**: If you set `autoFlushEventsInterval` to 0, you are responsible for calling `Amplify.Analytics.flushEvents()` to submit the recorded events to the backend.
 
+## Authentication events
+Indicate how frequently users authenticate with your application.
+
+On the **Analytics** page, the **Users** tab displays charts for **Sign-ins, Sign-ups, and Authentication failures**.
+
+To learn how frequently users authenticate with your app, update your application code so that Pinpoint receives the following standard event types for authentication:
+
+ - `_userauth.sign_in`
+ - `_userauth.sign_up`
+ - `_userauth.auth_fail`
+
+You can report authentication events by doing either of the following:
+
+- Managing user sign-up and sign-in with Amazon Cognito user pools.
```

Comment:
The json file should be `amplifyconfiguration.json` instead of `awsconfiguration.json` right?